### PR TITLE
Some yaml fix and cleanups after kube-batch in tree

### PR DESCRIPTION
**Pull request by hzxuzhonghu**

### CODE REVIEW

File: cmd/kube-batch/app/server.go [hunk 1]
```diff
@@ -72,7 +71,7 @@ func buildConfig(opt *options.ServerOption) (*rest.Config, error) {
 	return cfg, nil
 }
 
-// Run the kubeBatch scheduler
+// Run the kube-batch scheduler
```
Comment (k82cn) [inline cmd/kube-batch/app/server.go]: Should be volcano scheduler.

**Comment by hzxuzhonghu** [comment]
cc @TommyLike

File: cmd/kube-batch/app/server.go (hunk 1 above)
Comment (hzxuzhonghu) [reply cmd/kube-batch/app/server.go]: sorry, i missed this.

As there are many cleanup work left, let me open an issue

**Pull request merged**